vent/api/plugin_helpers.py

```diff
@@ -654,13 +653,19 @@ def start_containers(self,
                     if result[0]:
                         host = result[1]
                     else:
-                        # get the default device using netifaces
-                        # external library
-                        d_device = netifaces.gateways()
-                        d_device = d_device['default'][netifaces.AF_INET]
-                        host = netifaces.ifaddresses(d_device[1])
-                        host = host[netifaces.AF_INET]
-                        host = host[0]['addr']
+                        # now just requires ip, ifconfig
+                        route = check_output(('ip', 'route')).split('\n')
```

Comment:
i would put this whole thing in a try/except, if `ip route` fails this will crash on the split and subsequent code block